config/api.yml

```diff
@@ -3892,6 +3899,11 @@
       :delete:
       - :name: delete
         :identifier: vm_delete
+    :disks_subcollection_actions:
```

Comment:
@skateman you define this as `disks_subresource_actions` because as I understand you are not fetching collection of disk - then also specs will pass.

thanks